composer.json

```diff
@@ -17,14 +17,15 @@
         "php": ">=8.2",
         "oskarstark/enum-helper": "^1.5",
         "phpdocumentor/reflection-docblock": "^5.4",
+        "phpstan/phpdoc-parser": "^2.1",
         "psr/cache": "^3.0",
         "psr/log": "^3.0",
         "symfony/clock": "^6.4 || ^7.1",
         "symfony/http-client": "^6.4 || ^7.1",
         "symfony/property-access": "^6.4 || ^7.1",
         "symfony/property-info": "^6.4 || ^7.1",
         "symfony/serializer": "^6.4 || ^7.1",
-        "symfony/type-info": "^6.4 || ^7.1",
+        "symfony/type-info": "^7.2.3",
```

Comment:
do we really need 7.2.3 or can we use 7.2 here?

Reply:
yup, unfortunately

Reply:
Ok